src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write code here
+const inputs = document.querySelectorAll('input');
+
+for (const input of inputs) {
+  const label = document.createElement('label');
+
+  input.placeholder = input.name[0].toUpperCase()
+  + input.name.slice(1).split('N').join(' N');
```

Comment:
it's hard to read. add the right tabulation
```suggestion
  input.placeholder = input.name[0].toUpperCase()
    + input.name.slice(1).split('N').join(' N');
```